now.json

```diff
@@ -8,14 +8,9 @@
   },
   "routes": [
     {
-      "src": "/humidity",
+      "src": "/weather",
       "methods": ["GET"],
-      "dest": "/api/humidity.js"
-    },
-    {
-      "src": "/random",
-      "methods": ["GET"],
-      "dest": "/api/random.js"
+      "dest": "/api/weather.ts"
```

Comment:
I think this is right, but the extension might need to be `js` to reference the compiled file.

Reply:
Apparently this is ok? I'd still like to see documentation for it.